### PR TITLE
Fix #9638 ui issue with context when adding all plugins

### LINKED ISSUE #9638: Adding GeoProcessing and LongitudinalProfileTool plugins in a context will cause UI issue
## Description

you can have this issue if you include all plugins

## How to reproduce

- open this context https://dev-mapstore.geosolutionsgroup.com/mapstore/#/context/all-plugins

*Expected Result*

to have the burger menu rendered if both sidebar and burger menu are added to a context

*Current Result*

image

- [x] Not browser related

<details><summary> <b>Browser info</b> </summary>

(use this site: <a href="https://www.whatsmybrowser.org/">https://www.whatsmybrowser.org/</a> for non expert users)

| Browser Affected | Version |
|---|---|
|Internet Explorer| |
|Edge| |
|Chrome| |
|Firefox| |
|Safari| |
</details>

### PR DESCRIPTION
**Pull request by MV88**
## Description

fixed the issues in the contexts

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#9638

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Review by offtherailz — APPROVED** [review]
This menu is not so visible
image

So I pushed some changes to make it look like this:

image

From my testings now it works well. Please double check you too @MV88

**Review by offtherailz — CHANGES_REQUESTED** [review]
- [ ] with #/context/Longitudinal_profile
the overflow is now broken: 

image

Previous implementation worked correctly: 
image

- [ ] I fixed the menu style. Anyway when you close the burger menu, the popup stays open. We have to hide it. 
- [ ] Fix `useEffect`

**Review by offtherailz — APPROVED** [review]
Simplified logic behind menu

File: web/client/plugins/longitudinalProfile/Menu.jsx [hunk 1]
```diff
@@ -88,7 +89,32 @@ const Menu = ({
...
+                    <Glyphicon glyph="1-line"/>
+                    <Message msgId="longitudinalProfile.title"/>
+                </MenuItem></> :
+                <MenuItem active={menuIsActive || open} key="menu" onClick={() => setMenuOpen(!open)}>
+                    <Glyphicon glyph="1-line"/>
+                    <Message msgId="longitudinalProfile.title"/>
+                </MenuItem> }
+        </>);
+    } else {
+        MenuComp = DropDownMenu;
```
[the remark this answers is not in the document]
Comment (offtherailz) [reply web/client/plugins/longitudinalProfile/Menu.jsx]: Solved

**Comment by MV88** [comment]
@ElenaGallo please test it in DEV

**Comment by ElenaGallo** [comment]
Test passed, @tdipisa  it is possible to  marge the backport  https://github.com/geosolutions-it/MapStore2/pull/9652

**Pull request merged**